src/components/experiment-tracking/accordion/accordion.scss

```diff
@@ -27,6 +27,10 @@ $toggle-width: 1.15em;
   &--large {
     font-size: 1.6em;
   }
+
+  &--x-large {
+    font-size: 1.9em;
+  }
```

Comment:
Looking at the designs I don't see where it calls for this font size. Think we should be able to use what we already have.